stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MessageReplyView.kt

```diff
@@ -123,22 +123,22 @@ internal class MessageReplyView : FrameLayout {
     /**
      * Sets the background for message reply.
      *
-     * @param message [Message] The message replied message.
+     * @param quotedMessage [Message] The message contained in the reply bubble.
      * @param isMine Whether the message containing the reply is from the current user or not.
      * @param style [MessageReplyStyle] contains the styles of the background.
      */
-    private fun setReplyBackground(message: Message, isMine: Boolean, style: MessageReplyStyle?) {
+    private fun setReplyBackground(quotedMessage: Message, isMine: Boolean, style: MessageReplyStyle?) {
```

Comment:
Changed because the original naming was confusing and required more inspection to realize which message was the quoted part and which was the reply.